projects/components/src/select/select.component.ts

```diff
@@ -32,28 +32,38 @@ import { SelectSize } from './select-size';
       [ngClass]="[
         this.size,
         this.groupPosition,
-        selected ? selected.style.toString() : '',
         this.showBorder ? 'border' : '',
-        this.disabled ? 'disabled' : ''
+        this.disabled ? 'disabled' : '',
+        !this.multiSelectMode && selected ? selected.style.toString() : ''
       ]"
       *htcLetAsync="this.selected$ as selected"
     >
-      <htc-popover [disabled]="this.disabled" [closeOnClick]="true" class="select-container">
+      <htc-popover [disabled]="this.disabled" [closeOnClick]="!this.multiSelectMode" class="select-container">
         <htc-popover-trigger>
           <div class="trigger-content" [ngClass]="this.justifyClass">
             <htc-icon *ngIf="this.icon" class="trigger-prefix-icon" [icon]="this.icon" size="${IconSize.Small}">
             </htc-icon>
-            <htc-label class="trigger-label" [label]="selected?.label || this.placeholder"> </htc-label>
+            <htc-label
+              *ngIf="!this.multiSelectMode"
+              class="trigger-label"
+              [label]="selected?.label || this.placeholder"
```

Comment:
No real reason for this if/else right? the create trigger label logic supports both modes. It might be good to do it through na observable or save it too a var so we don't have to call this function each cycle.

Reply:
yea. no reason for if/else. I was only testing. will be cleaning this up